src/main/java/nextstep/ladder/LadderGame.java

```diff
@@ -12,9 +21,18 @@ public static void main(String[] args) {
         Users users = Users.from(UserGenerator.generateUsers(joinUsers));
         int countOfUser = users.size();
 
+        String ladderResult = InputView.inputLadderResult();
+        Results results = new Results(ladderResult);
```

Comment:
`ladderResult` 라는 지역변수는 선언되지 않고 바로 파라메터로 들어가도 될 것 같네요.

```suggestion
            Results results = new Results(InputView.inputLadderResult());

Reply:
네 바로 파라미터로 넘길 수 있는 값이라면 굳이 지역변수로 초기화 하고 사용하지 않아도 될 것 같네요 !